pkg/asset/agent/image/unconfigured_ignition.go

```diff
@@ -155,9 +148,9 @@ func (a *UnconfiguredIgnition) Generate(_ context.Context, dependencies asset.Pa
 		PullSecret:                pullSecretAsset.GetPullSecretData(),
 		ReleaseImages:             releaseImageList,
 		ReleaseImage:              clusterImageSet.Spec.ReleaseImage,
-		ReleaseImageMirror:        mirror.GetMirrorFromRelease(clusterImageSet.Spec.ReleaseImage, registriesConfig),
-		HaveMirrorConfig:          len(registriesConfig.MirrorConfig) > 0,
-		PublicContainerRegistries: getPublicContainerRegistries(registriesConfig),
+		ReleaseImageMirror:        "",
+		HaveMirrorConfig:          false,
+		PublicContainerRegistries: "",
```

Comment:
Same here, I'm not understanding why this was change or how is it related to removing the interactive disconnected workflow.